decentraland/AppNavigator.js

```diff
@@ -21,16 +47,43 @@ const AppNavigator = createStackNavigator(
   },
   {
     initialRouteName: "HomeScreen",
-    defaultNavigationOptions: {
-      headerStyle: {
-        backgroundColor: Colors.headerBackgroundColor,
+    defaultNavigationOptions,
+  }
+);
+
+const MyAssetsNavigator = createStackNavigator(
```

Comment:
This could potentially be just a screen rather than a navigator, unless we're using the stack functionality already

Reply:
We are using both navigators

Reply:
Ah ok - are there any screens that can appear on the stack on top of the default my assets screen?

Either way, I can certainly imagine that there might be in the future

Reply:
This won't gate merging but I'm still curious for your thoughts

Reply:
> Ah ok - are there any screens that can appear on the stack on top of the default my assets screen?
> 
> Either way, I can certainly imagine that there might be in the future

Sorry, I didn't get it.

Reply:
Right now there's a drawer navigator that opens either a list land stack (that starts on a list land screen and can send you through a stack that lets you buy land) or a my assets stack.

As far as I can tell, the my assets stack consists of only one screen, the home screen that shows a list of your assets.

For that reason, I think potentially the my assets stack doesn't need to be a stack navigator - it could just be a screen that is linked to from the drawer stack navigator

Reply:
Oh, got it now! Yes, I agree. I've done that because as a screen the header isn't showed and it seemed a good workaround (since this stack can have more screen in future).

Reply:
Ah ok, good to know. Yeah, I think there's probably another way around that problem, but that is a perfect motivation for going this route as a simple solution. Thanks for the info! Resolving this.